charts/kyuubi/templates/kyuubi-headless-service.yaml

```diff
@@ -25,7 +25,7 @@ spec:
   type: ClusterIP
   clusterIP: None
   ports:
-    {{- range $name, $frontend := .Values.server }}
+    {{- range $name, $frontend := merge .Values.server .Values.monitoring }}
```

Comment:
Similar to comment for `StatefulSet`, let's split rendering for server and monitoring ports.

Reply:
OK

Reply:
I will solve these problems tomorrow